packages/xo-web/src/xo-app/vm-import/utils.js

```diff
@@ -0,0 +1,12 @@
+import * as CM from 'complex-matcher'
+
+export const getRedirectionUrl = (vms = []) => {
+  const vmIds = typeof vms === 'object' ? Object.values(vms) : vms
+  return vmIds.length === 0
+    ? undefined // no redirect
+    : vmIds.length === 1
+    ? `/vms/${vmIds[0]}`
+    : `/home?s=${encodeURIComponent(
+        new CM.Property('id', new CM.Or(vmIds.map(vm => new CM.String(vm)))).toString()
+      )}&t=VM`
+}
```

Comment:
-  I have placed it here because I only use it for redirecting when importing VMs.
- It's specifically designed for redirecting VMs and cannot be used for other objects. We don't have the same URL for other objects.